src/css/homepage.css

```diff
@@ -1,3 +1,48 @@
+/*Homepage fonts*/
+h1 {
+	font-family: "Open Sans", Arial, sans-serif;
+	font-size: 25px;
+	color: white;
+}
+
+h3 {
```

Comment:
You're applying this to **all** `h3` tags in HTML, not only in the homepage, you should create either an id or class for this to be applied on the `h3`  tags.